package.json

```diff
@@ -45,15 +45,19 @@
     "lint-staged": "^7.2.0",
     "preact-cli": "^2.1.0",
     "preact-render-spy": "^1.2.1",
-    "prettier": "^1.13.5"
+    "prettier": "^1.13.5",
+    "react-modal": "^3.4.5"
   },
   "dependencies": {
     "babel-plugin-lodash": "^3.3.2",
     "c-3po": "^0.8.1",
+    "file-saver": "^1.3.8",
     "preact": "^8.2.6",
     "preact-cli-lodash": "^1.1.0",
     "preact-compat": "^3.17.0",
     "preact-router": "^2.5.7",
+    "react-copy-to-clipboard": "^5.0.1",
+    "u2f": "git+https://github.com/ProtonMail/u2f.git",
```

Comment:
Plz use this format `    "component-generator": "github:ProtonMail/componentGenerator#master",`  (_it seems to work better_) and move these 2 deps inside dependencies.